sql/2021/javascript/most_requested_content_type.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+  resp_content_type,
+  COUNT(0) AS count
+FROM
+  `httparchive.sample_data.summary_requests_*`
+GROUP BY
+  1
+ORDER BY
+  2 DESC
```

Comment:
1. Query the full dataset, `httparchive.almanac.requests` (`WHERE date = '2021-07-01'`)
2. Group by `client`
3. Do you want to restrict results to `WHERE type = script`?
4. Calculate a percent of total: `SUM(COUNT(0)) OVER (PARTITION BY client) AS total` which you can also use as the denominator to get the percent (similar to other queries)